heartbeat/oraacfs

```diff
@@ -0,0 +1,254 @@
+#!/bin/sh
+#
+#  License:      GNU General Public License (GPL)
+#  (c) 2020 Etzion Bar Noy
+#           and Linux-HA contributors
+#  Derived from oraacfs agent by O. Albrigtsen
+#
+# -----------------------------------------------------------------------------
+#      O C F    R E S O U R C E    S C R I P T   S P E C I F I C A T I O N
+# -----------------------------------------------------------------------------
+#
+# NAME
+#       oraacfs : OCF resource agent script for Oracle ASM Filesystem (ACFS)
+#
+
+# Initialization:
+: ${OCF_FUNCTIONS_DIR=${OCF_ROOT}/lib/heartbeat}
+. ${OCF_FUNCTIONS_DIR}/ocf-shellfuncs
+
+# Defaults
+OCF_RESKEY_user_default="grid"
+
+: ${OCF_RESKEY_user=${OCF_RESKEY_user_default}}
+
+
+oraacfs_usage() {
+	cat <<END
+    usage: $0 (start|stop|validate-all|meta-data|help|usage|monitor)
+    $0 manages a Oracle ACFS through Oracle GI commands
+    The 'start' operation mounts ACFS, or waits for ACFS to become available
+    The 'stop' operation stops the ACFS
+    The 'status' operation reports whether the ACFS is active
+    The 'monitor' operation is similar to status
+    The 'validate-all' operation reports whether the parameters are valid
+END
+}
+
+oraacfs_meta_data() {
```

Comment:
You should also add the user parameter to the metadata, so tools like `pcs` will accept using non-default user.

Reply:
The user was supposed to be used for 'srvctl' command, however, it can be called as root, with no ill effect.

Reply:
You have defined a default and made it possible to change it at:
https://github.com/ClusterLabs/resource-agents/pull/1459/files#diff-e70fcf13bc762e00fb3272abd7b02283R23

So it would make sense to add it to metadata as well.

Reply:
True. I have updated the file.
Thanks